packages/react-native-web/src/exports/Image/__tests__/index-test.js

```diff
@@ -174,6 +173,38 @@ describe('components/Image', () => {
       expect(onLoadEndStub.mock.calls.length).toBe(2);
     });
 
+    test('is called on update if "headers" are modified', () => {
+      const onLoadStartStub = jest.fn();
```

Comment:
It looks like this stub is never looked at in the assertions, is it necessary to have or can it be removed?

Reply:
OK, I think just remove it then (or add an assertion that does something). I don't think in this PR we need to be worried about maintaining consistent test structure with what someone else is doing (especially when we don't care what they're doing).